src/organizations/components/OrgProfileTab/DeletePanel.tsx

```diff
@@ -65,6 +65,7 @@ const DeleteOrgButton: FC = () => {
   const orgCanBeSuspended = useSelector(selectOrgSuspendable)
   const canCreateOrgs = useSelector(selectOrgCreationAllowance)
   const {users} = useContext(UsersContext)
+  const {account} = useSelector(selectCurrentIdentity)
 
```

Comment:
You can use a `selectCurrentAccount` selector here, to be more specific. There is potential for a re-render because `identity.org` gets more data loaded into it on one or two pages.